app/src/main/res/layout/main_activity.xml

```diff
@@ -1,7 +1,19 @@
 <?xml version="1.0" encoding="utf-8"?>
-<FrameLayout xmlns:android="http://schemas.android.com/apk/res/android"
-    xmlns:tools="http://schemas.android.com/tools"
-    android:id="@+id/container"
-    android:layout_width="match_parent"
-    android:layout_height="match_parent"
-    tools:context=".MainActivity" />
+<RelativeLayout xmlns:android="http://schemas.android.com/apk/res/android"
```

Comment:
layout is not necessary for now.  just use fragment in the root. also prefer ConstraintLayout in the future for this purposes.